bolt/include/bolt/Profile/DataAggregator.h

```diff
@@ -444,6 +464,35 @@ class DataAggregator : public DataReader {
   /// an external tool.
   std::error_code parsePreAggregatedLBRSamples();
 
+  /// Dump pre-parsed perf profile data into a single file.
+  /// The generator relies on the aggregator work to spawn the required
+  /// perf-script jobs based on the the aggregation type, and merges
+  /// the results of the pref-script jobs into a single file.
+  /// This hybrid profile contains all required events such as BuildID,
+  /// MMAP, TASK, Branch/BrStack, or Memory for the aggregation.
```

Comment:
```suggestion
  /// MMAP, TASK, MAIN (brstack or basic samples), OR MEM for the aggregation.
```


Also can you expand a bit around those events? ie which ones we expect to be always there (mmaps, task, main) and which ones are optional? And that MAIN is either basic-events or brstack?